### ISSUE TITLE
[bug] Clicking 'Fetch Metadata' hangs the application

### ISSUE THREAD

**Issue by ak-42** [crocodilestick/Calibre-Web-Automated#127]
**Describe the bug**
Clicking 'Fetch Metadata' for a book hangs the application, necessitating container restart.

**To Reproduce**
1. Open the application
2. Select any book
3. Click 'Edit Metadata'
4. Click 'Fetch Metadata'
5. 'Fetch Metadata' window appears, and hangs on 'Loading' and eventually says 'Serch Error'
6. Application no longer responsive on browser refresh or in another tab (container restart required)

**Expected behavior**
The metadata is fetched and offered to the user.

**Screenshots**
Nothing informative appears in the 'Fetch Metadata' windows, so I won't screenshot that.

**Configuration(please complete the following information):**
 - OS: Docker Compose
 - Hardware: x86 N100 machine, 16GB RAM
 - Version :latest tag

**Additional context**
Neither the calibre-web logs (on DEBUG level), nor the container logs themselves show anything relevant/out of ordinary, which is why I'm not attaching them.

Presents similarly to janeczku/calibre-web#2562, but in this case as best as I can tell, the API is not down (and it doesn't even let me unselect APIs as the users in that issue suggest).

**Comment by mckenna654** [comment]
Also getting this issue

**Comment by swotai** [comment]
I have the same issue. I did a bit of hack around and debug by bashing into the container and overwriting the calibre-web to 0.6.24, and the metadata search started working again. Checking the docker file i see that it's hardcoded to 0.6.23 [dockerfile arg line](https://github.com/crocodilestick/Calibre-Web-Automated/blob/5bbd15af623d02d4ea6e724df43efc314b88ffad/Dockerfile#L28). Recommend to set it to 24 to fix this issue. Also wondering why not let it be blank and let the build process just pick latest?

**Comment by kalpik** [comment]
Indeed. This is an issue with calibre-web release. The changelog there also says it fixed some search results from Amazon and Google. If you uncheck these two, the find metadata starts working. Hoping for a quick fix for this one.

**Comment by kalpik** [comment]
Created a PR: https://github.com/crocodilestick/Calibre-Web-Automated/pull/146

**Comment by ak-42** [comment]
@crocodilestick Hi. I am still seeing this issue in version V3.0.1. It presents exactly the same.

**Comment by giannihill** [comment]
Also getting this error in V3.0.1.

**Comment by kalpik** [comment]
For what it's worth, I don't have this issue anymore.

**Comment by JDIacobbo** [comment]
I am experiencing this issue in 3.0.4.

**Comment by platcham** [comment]
I am also experiencing this bug on 3.0.4. Fetching metadata on any book in my library instantly freezes the container until I restart it. Running through Unraid if that matters.

**Comment by JCandle** [comment]
Same as @platcham. @crocodilestick

**Comment by raphaelgrau** [comment]
I am also experiencing this issue.
@yarafie any idea?

**Comment by yarafie** [comment]
> I am also experiencing this issue. [@yarafie](https://github.com/yarafie) any idea?

Wish I could help but it is working for me just checked.

**Comment by daften** [comment]
Same issue on 3.0.4 with calibre-web 0.6.24 using a linuxserver docker image on synology nas (using docker-compose).

**Comment by apurva-shukla** [comment]
Having this issue with V3.1.2 as well. Using a docker container on proxmox to run this. The container has enough CPU and RAM which I previously believed was the issue.
